src/components/pages/Drafts/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect, UIEvent } from 'react';
+import { RootState } from 'src/redux';
+import Table from 'react-bootstrap/Table';
+import { bindActionCreators, Dispatch } from 'redux';
+import { connect, ConnectedProps } from 'react-redux';
+import './index.css';
+import { Container, Spinner } from 'react-bootstrap';
+import { loadLetterDrafts } from 'src/redux/modules/volunteer';
+
+const mapStateToProps = (state: RootState) => ({
+  user: state.user,
+  drafts: state.volunteers.drafts,
+});
+
+const mapDispatchToProps = (dispatch: Dispatch) =>
+  bindActionCreators({ loadLetterDrafts }, dispatch);
+
+const connector = connect(mapStateToProps, mapDispatchToProps);
+type PropsFromRedux = ConnectedProps<typeof connector>;
+
+const UnconnectedDrafts: React.FC<PropsFromRedux> = ({
+  loadLetterDrafts,
+  user,
+  drafts,
+}) => {
+  const [hasFetchedData, setHasFetchedData] = useState<boolean>(false);
+
+  const { token } = user.user;
+  const { org } = user.user;
+
+  useEffect(() => {
+    if (!hasFetchedData && org) {
+      loadLetterDrafts(token, org.id);
+      setHasFetchedData(true);
+    }
+  });
+
+  const spinner = (
+    <Container id="contacts-spinner">
+      <Spinner animation="border" role="status" variant="primary">
+        <span className="sr-only">Loading...</span>
+      </Spinner>
+    </Container>
+  );
+
+  let page_id = 'content';
+
+  const handleScroll = (e: UIEvent) => {
+    const tableDiv = document.getElementById('tableDiv');
```

Comment:
what is this doing ?

Reply:
This is how we have the pagination of contacts happening - it detects when the user hits the bottom of the screen and then loads more contacts